### ISSUE TITLE
Erratic action bar transition when using bound (or empty) title property

### ISSUE THREAD

**Issue by ignaciofuentes** [NativeScript/NativeScript#2672]
### Tell us about the problem

Erratic action bar transition when using bound (or empty) title property on the Actionbar that is being navigated to
### Which platform(s) does your issue occur on?

iOS
### Please provide the following version numbers that your issue occurs with:

2.2.1
### Please tell us how to recreate the issue in as much detail as possible.

When setting the title of an actionBar to be dataBound the incoming actionBar navigates from the middle of the page up.
Please see [this screencast.](http://screencast.com/t/aXYs6XEmeLNA)
My current workaround is to set the details-page to have actionBarHidden set to false.
The same thing happens when there is no ActionBar definition at all in the incoming page. The ActionBar eventually is on the page, but the transition does the same odd middle-up sliding
### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.

details-page.xml

```
<Page xmlns="http://schemas.nativescript.org/tns.xsd" navigatingTo="onNavigatingTo">
  <Page.actionBar>
      <ActionBar title="{{title}}"/>
  </Page.actionBar>
  <StackLayout>
    <Label text="Second page" class="title"/>  
  </StackLayout>
</Page>
```

details-page.js

```
function onNavigatingTo(args) {
    var page = args.object;    
    page.bindingContext = {title:"Dynamic title"};
}
exports.onNavigatingTo = onNavigatingTo;
```

**Comment by tsonevn** [comment]
Hi @ignaciofuentes,

Thank you for reporting this issue.

I was able to reproduce this problem with `ActionBar` transition in iOS. In regard to that I am attaching  [link](https://github.com/tsonevn/ActionBarTransition) to a sample project where this issue has been reproduced. We will investigate this problem and will provide fix in some of the next NatioveScript versions.

Regards,
@tsonevn

**Comment by tzraikov** [comment]
Will be addressed with #2870
